stackdriver/client.go

```diff
@@ -82,6 +82,16 @@ type recoverableError struct {
 // version.* is populated for 'promu' builds, so this will look broken in unit tests.
 var userAgent = fmt.Sprintf("StackdriverPrometheus/%s", version.Version)
 
+var counter = 0
+var ips = []string{"199.36.153.4:443", "199.36.153.5:443",
```

Comment:
Please move ":443" to the DialContext call below, otherwise this isn't really an IP address.